### DISCUSSION TITLE
请问ReactHookForm如何与useForm结合使用

### DISCUSSION THREAD

**Discussion by Engimaz** [alovajs/alova#586] [Q&A] [answered]
我从官网文档中并没有发现有很好的例子来处理校验逻辑，我打算使用reacthookform结合使用。使用他的叫校验方法与useForm的缓存功能。应该如何使用

**Comment by MeetinaXD** [comment, marked as answer]
你有什么特殊的需求吗？按照正常的使用方式操作就可以了，先校验再 submit。
